gulpfile.babel.js

```diff
@@ -81,49 +74,30 @@ gulp.task('lint', () => {
 
 gulp.task('build', () => {
   return gulp.src('src/*.js')
-          .pipe(errorNotifier())
-
-            .pipe(gulpif(argv.debug, sourcemaps.init()))
-            .pipe(rollup(OPTIONS.rollup))
+    .pipe(errorNotifier())
 
-          .pipe(gulp.dest('.'))
+      .pipe(processJs())
 
-            // Minify and pipe out
-            .pipe(gulpif(!argv.debug, uglify(OPTIONS.uglify)))
-            .pipe(gulpif(argv.debug, sourcemaps.write()))
-            .pipe(rename(path => path.extname = '.min.js'))
-            .pipe(size({ gzip: true }))
+    .pipe(gulp.dest('.'))
 
-          .pipe(gulp.dest('.'))
-          .pipe(notify('build finished'));
```

Comment:
And if that's the case, wouldn't a much better solution be to make a gulp task to refresh the tests for you when files change? That's kind of the point of a `watch`?